wasmbinding/bindings/msg.go

```diff
@@ -17,6 +19,8 @@ type NeutronMsg struct {
 	RegisterInterchainQuery   *RegisterInterchainQuery   `json:"register_interchain_query,omitempty"`
 	UpdateInterchainQuery     *UpdateInterchainQuery     `json:"update_interchain_query,omitempty"`
 	RemoveInterchainQuery     *RemoveInterchainQuery     `json:"remove_interchain_query,omitempty"`
+	AddAdmin                  *AddAdmin                  `json:"add_admin,omitempty"`
```

Comment:
why exactly do we need an add admin message available for the contracts? aren't we going to add a single admin to genesis?

Reply:
That was a very first method i implemented, so i have not removed `AddAdmin` exactly to leave a space to discussion - "are we need this one?".